### DISCUSSION TITLE
Turn off auto-update feature

### DISCUSSION THREAD

**Discussion by khoinhtvn** [Security-Onion-Solutions/securityonion#11885] [2.4] [answered]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

16

### Storage for /

85GB

### Storage for /nsm

166GB

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Dear Security Onion Community,

I would like to know how to turn off auto-update such as for OS Updates. I could not find the configuration option while installing Security Onion Manager. 

Thank you for your support!

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by Mav1814** [comment, marked as answer]
Hi, 
If i'm not mistaken go to Administration -> Configuration -> patch -> os -> enable -> off.

cheers
mav.

**Reply by khoinhtvn** [reply]
Hi Mav,

I found the setting you mentioned and it works.

I set it to false. Then, I no longer see the information about OS update when executing "salt-call schedule.list".

I should have checked the UI more carefully. I have been spending time looking for config files. 

Thank you so much for your help!

Regards,
Khoi
